service/src/main/java/org/cbioportal/genome_nexus/service/annotation/VariantClassificationResolver.java

```diff
@@ -132,7 +132,7 @@ private String resolveVariantClassification(String variant, String variantType,
     {
         String defaultValue = "Targeted_Region";
 
-        if (variant == null || variantType == null) {
+        if ((variant == null || variantType == null) && !variant.contains("splice")) {
```

Comment:
This will definitely produce a null pointer exception when variant is null. Do we want to return `null` if `variant != null && variant.contains("splice")`? Or do we want to return `null` if `variant == null || variantType == null || variant.contains("splice")`? Or is it something else? :)

Reply:
@onursumer doh - good catch fixed